crates/locutus-core/src/client_events/websocket.rs

```diff
@@ -42,13 +46,13 @@ impl WebSocketProxy {
     where
         T: Into<SocketAddr>,
     {
-        let filter = warp::filters::path::end().map(warp::reply::reply).boxed();
+        let filter = Router::default();
```

Comment:
Change mentions of filter to `route` everywhere.

Reply:
Name changed to router